testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeAlterTableCompatibility.java

```diff
@@ -443,9 +441,7 @@ public void testIdentityColumn()
                     .contains("b BIGINT GENERATED ALWAYS AS IDENTITY");
             onDelta().executeQuery("INSERT INTO default." + tableName + " (a) VALUES (0)");
 
-            DatabricksVersion databricksRuntimeVersion = getDatabricksRuntimeVersion().orElseThrow();
-            // Actual value for IDENTITY column varies between Databricks versions
-            QueryAssert.Row expected = databricksRuntimeVersion.isOlderThan(DATABRICKS_122_RUNTIME_VERSION) ? row(0, 1) : row(0, 2);
```

Comment:
`0, 2`  no longer possible?
or is it just randomized?

Reply:
`0, 2` happened on the latest master though it was `0, 1` past few days. I changed it to allow both values.

Reply:
It was always `0, 1` the past few days. Let me update to allow both values if this issue happens next time.